orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/model/DeliveryConfig.java

```diff
@@ -0,0 +1,100 @@
+package com.netflix.spinnaker.orca.front50.model;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DeliveryConfig {
```

Comment:
I suspect we have `lombok` kicking around and you could `@Data` this and save some typing.